connectors/s3-connector/src/test/scala/zio/connect/s3/LiveS3ConnectorSpec.scala

```diff
@@ -5,12 +5,14 @@ import org.testcontainers.utility.DockerImageName
 import software.amazon.awssdk.auth.credentials.{AwsBasicCredentials, StaticCredentialsProvider}
 import software.amazon.awssdk.regions.Region
 import zio.aws.core.config.AwsConfig
+import zio.aws.core.httpclient.HttpClient
 import zio.aws.netty.NettyHttpClient
 import zio.aws.s3.S3
+import zio.test.Spec
 import zio.{Scope, ZIO, ZLayer}
 
 object LiveS3ConnectorSpec extends S3ConnectorSpec {
-  override def spec =
+  override def spec: Spec[Scope, Object] =
```

Comment:
I think it's better if we don't set the type here. I find it easier to work with specs without setting it.